eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -2,6 +2,7 @@ parameters:
   Repos: []
   DirectoryToSync: eng/common
   CommitMessage: commit-message-not-set
+  PRBranchName: branch-name-not-set
 
 steps:
 
```

Comment:
For now lets start with this simple logic of blindly copying over the directory but at some point we might need to get a little more scoped here and only include changes as part of the tools PR.